### DISCUSSION TITLE
网页记录一天使用了50个小时……

### DISCUSSION THREAD

**Discussion by monkey-sking** [Planshit/Tai#252] [General]
![image](https://github.com/Planshit/Tai/assets/5218113/395749c0-b7e2-4989-87eb-af0d5d91bf9d)

如图，一天记录了50个小时……

**Comment by noberumotto** [comment]
麻烦将当日的log提供一下，littlepandause@gmail.com

**Reply by monkey-sking** [reply]
已发送

**Reply by noberumotto** [reply]
收到
